account_invoice_force_removal/__manifest__.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2020 Tecnativa - Víctor Martínez
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+{
+    "name": "Account Invoice Force Removal",
+    "summary": """Allow force removal invoice""",
+    "version": "13.0.1.0.0",
+    "category": "Account",
+    "license": "LGPL-3",
```

Comment:
We use LGPL only for website modules.

```suggestion
    "license": "AGPL-3",
```

You need to change all header comments.